src/main/java/org/gradle/profiler/studio/launcher/StudioConfigurationProvider.java

```diff
@@ -59,6 +63,10 @@ private static StudioConfiguration getMacOSConfiguration(Path studioInstallDir)
         Map<String, String> systemProperties = mapValues(jvmOptions.dict("Properties").toMap(), v -> v.replace("$APP_PACKAGE", actualInstallDir.toString()));
         String mainClass = jvmOptions.string("MainClass");
         Path javaCommand = actualInstallDir.resolve("Contents/jre/Contents/Home/bin/java");
```

Comment:
Can you add the possible paths to a list and then find the first one which exits? And if none of the paths exists, does that mean that the build should fail?

Reply:
Yes, I can do that. I think we can fail in that case. I don't see a way that it would work without the valid path